code/modules/clothing/spacesuits/captain.dm

```diff
@@ -1,15 +1,30 @@
 //Captain's Spacesuit
 /obj/item/clothing/head/helmet/space/capspace
 	name = "space helmet"
-	icon_state = "capspace"
+	icon_state = "capspace_0"
 	item_state = "capspacehelmet"
 	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Only for the most fashionable of military figureheads."
 	body_parts_covered = HEAD|EARS|EYES
 	permeability_coefficient = 0.01
 	pressure_resistance = 200 * ONE_ATMOSPHERE
 	armor = list(melee = 65, bullet = 50, laser = 50,energy = 25, bomb = 50, bio = 100, rad = 50)
+	light_power = 1.7
+	var/brightness_on = 4
+	var/on = 0
+	action_button_name = "Toggle Helmet Light"
 	species_fit = list(VOX_SHAPED)
 
+/obj/item/clothing/head/helmet/space/capspace/attack_self(mob/user)
+	on = !on
+	icon_state = "capspace_[on]"
+	user.update_inv_head()
+
+	if(on)
+		set_light(brightness_on)
+	else
+		set_light(0)
+		user.update_inv_head()
```

Comment:
The whole thing should probably be a ternary but I'm not gonna make him rewrite helmets for it.